config/create.remote.sample-AWS.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "aws-testnet",
+  "type": "aws",
+  "nodes": 1,
+  "nonValidatorIndices": [0],
+  "environmentNodeKeys": true,
+  "environmentKeys": true,
+  "keep": true,
+  "resources": {
+    "requests": {
+      "memory": "9Gi",
+      "cpu": "1.5"
+    }
+  },
+  "remote": {
+    "monitoring": true,
+    "projectID": "polkadot-benchmarks",
```

We should use a generic project name, not our own.
```suggestion
    "projectID": "my-gcp-project-id",
```